src/utils/patient-identifier-helper.ts

```diff
@@ -11,8 +11,8 @@ export const saveIdentifier = (patient: fhir.Patient, patientIdentifier: Patient
 };
 
 export const getPatientLatestIdentifier = (patient: fhir.Patient, identifierType: string) => {
-  const patientIdentifiers = patient.identifier;
-  return patientIdentifiers.find((identifier) => {
+  const patientIdentifiers = patient?.identifier;
+  return patientIdentifiers?.find((identifier) => {
```

Comment:
@gitcliff do we need the `patientIdentifiers` state?

```suggestion
  return patient?.identifier?.find((identifier) => {
```

Reply:
Plus is it possible for the patient to null/undefined?